### PR TITLE
[DOCS] Align with new TYPO3 documentation standards

### PR DESCRIPTION
**Pull request by alexander-nitsche**
Relates: https://github.com/TYPO3-Documentation/T3DocTeam/issues/182

### CODE REVIEW

**Comment by alexander-nitsche** [comment]
This is how it looks like:
Screen Shot 2022-04-02 at 10 49 23

**Comment by alexander-nitsche** [comment]
Added two new commits regarding badges in README and additional sources (TER, repository and docs.typo3.org) in composer.json.

In addition, the TER homepage is missing the "Code Insights", "Found an issue?" and "Packagist" buttons, which seem to be caused by the empty "Link to repository", "Link to issue tracker", "Composer name of extension" fields in the TER extension edit form.

Relates: TYPO3-Documentation/T3DocTeam#185

File: Documentation/Contribution/Index.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+We will publish a new version every once in a while, depending on the amount of
+changes and pull requests submitted.
+
+Authors
+-------
+
+Over a dozen contributors have been working on this area while this
+functionality was part of the TYPO3 Core. This package is now maintained by a
+loose group of TYPO3 enthusiasts inside the TYPO3 Community. Feel free to
+contact `Benni Mack`_ for any questions regarding "dbal".
```
Comment (ohader) [inline Documentation/Contribution/Index.rst:19]: Please avoid pointing to Benni here, refer to the issue tracker instead...
(seems that basically all legacy extensions had that, I wonder why...)

**Review by ohader — CHANGES_REQUESTED** [review]
Avoid author pointing...

**Comment by ohader** [comment]
Styles CI & Travis CI did not respond back, probably can be dropped (at least Travis)

File: Documentation/Contribution/Index.rst (hunk 1 above)
Comment (alexander-nitsche) [reply Documentation/Contribution/Index.rst]: Done.

**Review by ohader — APPROVED** [review]

**Comment by ohader** [comment]
TER foo done

**Pull request merged**